core/templates/dev/head/domain/exploration/OutcomeObjectFactorySpec.js

```diff
@@ -25,19 +25,19 @@ describe('Outcome object factory', function() {
 
   it('should correctly determine if an outcome is confusing given a ' +
```

Comment:
Probably should break after '(' since the contents of the paren span more than one line?

Reply:
In the meantime shall we break it? Odd for the function() two lines below to have the same indentation level as it().

Reply:
Yes, that sounds greate! I will take care of it.